base_export_manager/static/src/js/base_export_manager.js

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2019 brain-tec AG (Brig, Switzerland, http://www.braintec-group.com)
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+
+odoo.define('base_export_manager.base_export_manager', function (require) {
+    'use strict';
+
+    const core = require("web.core");
+    const Sidebar = require("web.Sidebar");
+    const session = require("web.session");
+    const _t = core._t;
+
+    Sidebar.include({
+        _addItems: function (sectionCode, items) {
+            let _items = items;
+            const render_export_enalble = jQuery.inArray(
+                this.env.model, session.export_models);
+            if (!session.is_superuser && sectionCode === 'other' &&
+                    items.length && render_export_enalble < 0) {
+                _items = _.reject(_items, {label:_t("Export")});
+            }
+            this._super(sectionCode, _items);
+        },
+    });
+});
```

Comment:
This file does not show technical name...
V12.0
![Selección_588](https://user-images.githubusercontent.com/7689807/77967996-92d99900-72e6-11ea-8c35-6d52ec3c9484.png)
V13.0
![Selección_589](https://user-images.githubusercontent.com/7689807/77968022-9cfb9780-72e6-11ea-8b67-e9ed4f27786e.png)
See https://github.com/odoo/odoo/blob/43a115e94bea04a0533e57fb3bc1e52571a74c0b/addons/web/static/src/js/widgets/data_export.js#L76